README.md

```diff
@@ -1,33 +1,54 @@
 # hedera-solidity-deployer
 
-> Deploy Solidity Contracts to Hedera Locally using JavaScript.
+Deploy Solidity Contracts to Hedera Testnet Locally using JavaScript.
 
 ## Get Started
 
 ### Clone
 
-First, clone this project
+The first step is to clone this project
 
 ```
 git clone git@github.com:pieFi-platform/hedera-solidity-deployer.git
 ```
 
-then enter the directory
+and then enter the directory
 
 ```
 cd hedera-solidity-deployer
 ```
 
-### Install dependencies
+### Install Dependencies
+
+Run this command to install the dependencies
 
 ```
 npm install
 ```
 
-### Create .env file
+## Create Smart Contract
+
+Once the project has been created, the next step is to create the smart contract that we want to deploy. To create the contract file, create a .sol file in the src/contracts folder
+
+```
+touch ./src/contracts/SmartContractExample.sol
+```
 
-The .env file will store all of the variables needed to create the
-file on Hedera and deploy the smart contract.
+Now right your smart contract in this file using Solidity.
+
+### Compile Smart Contract to Bytecode
+
+Once your contract is written, it'll first need to be compiled to bytecode before being deployed. The following command will compile the smart contract and place it in the src/contracts/bin folder
+
+```
+solcjs --bin ./src/contracts/SmartContract.sol -o ./src/contracts/bin
```

Comment:
On line 34 above the contract is called SmartContractExample.sol. Here it is SmartContract.sol